esp-ieee802154/src/raw.rs

```diff
@@ -143,7 +142,6 @@ fn ieee802154_mac_init() {
     // memset(s_rx_frame, 0, sizeof(s_rx_frame));
     // s_ieee802154_state = IEEE802154_STATE_IDLE;
 
-    interrupt::enable(Interrupt::ZB_MAC, Priority::Priority1).unwrap();
     unsafe {
         esp_hal::riscv::interrupt::enable();
     }
```

Comment:
Handlers are not automatically registered anymore in esp-hal 0.17 so we'd need this here:
```rust
    unsafe {
        esp_hal::interrupt::bind_interrupt(
            esp_hal::peripherals::Interrupt::ZB_MAC,
            ZB_MAC.handler(),
        );
    }
    esp_hal::interrupt::enable(esp_hal::peripherals::Interrupt::ZB_MAC, ZB_MAC.priority()).unwrap();
```